lib/ts/recipe/passwordless/utils.ts

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { normaliseAuthRecipe } from "../authRecipe/utils";
+import { defaultEmailValidator } from "../emailpassword/validators";
+import { Config, NormalisedConfig, PasswordlessNormalisedBaseConfig, PasswordlessFeatureBaseConfig } from "./types";
+import { RecipeInterface } from "./types";
+import { defaultPhoneNumberValidator } from "./validators";
+
+export function normalisePasswordlessConfig(config: Config): NormalisedConfig {
+    const override: any = {
+        functions: (originalImplementation: RecipeInterface) => originalImplementation,
+        components: {},
+        ...config.override,
+    };
+
+    return {
+        ...normaliseAuthRecipe(config),
+        resendCodeTimeGap: config.resendCodeTimeGap === undefined ? 900 : config.resendCodeTimeGap,
```

Comment:
COnsider renaming the config to something like `resendCodeTimeGapInSeconds` or whatever the time interval is